src/Backend/Modules/ContentBlocks/Actions/Delete.php

```diff
@@ -31,19 +32,8 @@ public function execute(): void
         }
         $deleteFormData = $deleteForm->getData();
 
-        /** @var ContentBlockRepository $contentBlockRepository */
-        $contentBlockRepository = $this->get('content_blocks.repository.content_block');
-
-        $contentBlock = $contentBlockRepository->findOneByIdAndLocale(
-            $deleteFormData['id'],
-            Locale::workingLocale()
-        );
-
-        if ($contentBlock === null) {
-            $this->redirect(BackendModel::createURLForAction('Index', null, null, ['error' => 'non-existing']));
-
-            return;
-        }
+        /** @var ContentBlock $contentBlock */
```

Comment:
Should have been handled in https://github.com/forkcms/forkcms/pull/2096 - but fixed c0543e2e1